libraries/AP_GPS/AP_GPS.h

```diff
@@ -139,6 +140,7 @@ class AP_GPS
         Location location;                  ///< last fix location
         float ground_speed;                 ///< ground speed in m/sec
         float ground_course;                ///< ground course in degrees
+        float gps_yaw;                      ///< GPS derived yaw information, if available
```

Comment:
Is this in degrees or radians? (Should be in the comment)

Reply:
fixed